resources/analysisTools/indelCallingWorkflow/confidenceAnnotation_Indels.py

```diff
@@ -534,7 +534,8 @@ def main(args):
             entries[2] = dbsnp_id + "_" + dbsnp_pos
 
         ### Change the columns to make sure control is always in column 9 and tumor in column 10 (0 based)
-        entries[9], entries[10] = entries[header_indices["CONTROL_COL"]], entries[header_indices["TUMOR_COL"]]
+        if not args.no_control:
+            entries[9], entries[10] = entries[header_indices["CONTROL_COL"]], entries[header_indices["TUMOR_COL"]]
```

Comment:
But is this really correct? Should we really **not** switch if there is no control? Is it certain that `entries[10]` will always be the tumor, if there is no control? Otherwise this code may result in the tumor-value still be in `entries[9]`. 

Maybe the following would be correcter:

```python
entries[9], entries[10] = \
   entries.get(header_indices["CONTROL_COL"], None),\
   entries[header_indices["TUMOR_COL"]]
```

* Assumptions
  * The control value should be "None" (not sure whether that is correct)
  * `header_indices["CONTROL_COL"]` does not fail with an error, in no-control case (also not sure).

Whether these assumptions hold or not, the idea would be to always switch, but use appropriate values, instead of not switching in the no-control case.

Does that make sense?

Reply:
The switch only happens when we have two samples, where we always want control to be first and the tumor next. But for nocontrol workflows, the tumor is the only sample, so it will be the first column after the default VCF columns (#CHROM:FORMAT), which is entries[9]. So we don't need a switch for the nocontrol.

Reply:
Although I have to admit that it is kind of confusing that the tumor goes into the column that usually is the control. Or will the control column be missing completely, such that the table has one column less?

Reply:
Yes, there will be one less column. If we compare the default and nocontrol workflows, it is confusing, but this is how both SNVs and indel nocontrol workflows are configured.

Reply:
O.k. that's fine. If it is less columns I don't find it confusing, actually. Having a column full of None values would be fine and be more explicit, but changing the structure of the file is also fine.